plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/monitor/models/HuaweiMonitorMetric.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.orchestrator.plugin.huaweicloud.monitor.models;
+
+/**
+ * Huawei Monitor Metric.
+ */
+public class HuaweiMonitorMetric {
+
+    //cpu
+    public static final String CPU_USAGE = "cpu_usage";
+    public static final String CPU_UTIL = "cpu_util";
```

Comment:
should be full name.. CPU_UTILIZED and MEM_UTILIZED.

If possible these metrics can be a enum also.